### DISCUSSION TITLE
New Affilaite limits updates

### DISCUSSION THREAD

**Discussion by bf-mike-mkt** [betfinio/public#114] [Bugs and Reports]
hey please update new limits on following places:

1. Unlock academy courses minimum stake LOGIC: from 120k to 80k
2. Webinars unlock logic from 120k to 80k
3. update academy text: 
A https://betfin.network/academy/lesson/2/9 (from 60k to 40k) + replace screenshot
B https://betfin.network/academy/lesson/2/8 from (120k to 80k) + replace screenshot
C https://betfin.network/academy/lesson/22/68 update minimum from 120k to 80k

Please upadate as well czech version of academy

**Comment by bf-gabagool** [comment]
The limits as well as screenshots has been updated in both English and Czech language. Thank you for letting us know!
